docs/docs/examples/evaluation/batch_eval.ipynb

```diff
@@ -136,22 +136,22 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install spacy datasets span-marker scikit-learn"
+    "%pip install spacy datasets span-marker scikit-learn"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "id": "e31e10e6",
    "metadata": {},
    "outputs": [
     {
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "/home/loganm/llama_index_proper/llama_index/llama_index/evaluation/dataset_generation.py:187: DeprecationWarning: Call to deprecated class DatasetGenerator. (Deprecated in favor of `RagDatasetGenerator` which should be used instead.)\n",
+      "/Users/yi/Code/llama/llama_index/llama-index-core/llama_index/core/evaluation/dataset_generation.py:212: DeprecationWarning: Call to deprecated class DatasetGenerator. (Deprecated in favor of `RagDatasetGenerator` which should be used instead.)\n",
```

Comment:
@logan-markewich so if I use RagDatasetGenerator it only lets me specify how many generations per chunk but not the number of generations total. So here it would be >10 questions/answers.

Reply:
Resolving. Talked to Logan and we'll just leave the deprecation warning for now.